cdprofiles_build/build.sh

```diff
@@ -9,12 +9,11 @@ display "Loading crime data"
 docker run --rm\
     -v $(pwd):/src\
     -w /src/python\
-    --user $UID\
     -e API_TOKEN=$API_TOKEN\
     -e BUILD_ENGINE=$BUILD_ENGINE\
     -e V_CRIME=$V_CRIME\
     python:3.7-slim bash -c "
-        pip3 install -q pff-factfinder;
+        pip3 install -q requests;
```

Comment:
pff-factfinder comes with requests as a dependency, the reason why the installation failed is not because we specified the wrong dependency, but because of the --user flag failed all pip installations. 

to make things easier and more standardized we should have something like
```bash
docker run --rm\
    -v $(pwd):/src\
    -w /src/python\
    --user $UID\
    -e API_TOKEN=$API_TOKEN\
    -e BUILD_ENGINE=$BUILD_ENGINE\
    --env-file=version.env\
    python:3.7-slim bash -c "
        pip3 install -q pff-factfinder;
        python3 $1" |
psql $BUILD_ENGINE -f $2 
```
all the poverty, acs, decennial version var can be passed through `version.env`